winui/ComboBox/Searching.md

```diff
@@ -49,7 +49,7 @@ comboBox.TextMemberPath = "ID";
 
 For e.g. After typing `4` in selection box.
 
-![WinUI ComboBox text searching based on TextMemberPath](Searching_images/winui-combobox-textmemberpath-searching.png)
+![TextMemberPath searching editable mode in WinUI ComboBox](searching_images/winui-combobox-textmemberpath-searching-editable-mode.png)
```

Comment:
Revert your changes in all places. Existing alt text and image fine seems fine, change the casing  in Image Folder name.

Reply:
Reverted